### DISCUSSION TITLE
Reload dropdown's remote content?

### DISCUSSION THREAD

**Discussion by hakeem86** [fomantic/Fomantic-UI#1931] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
How do I reload dropdown content? 
I tried with refresh but that does not refresh content ? 

$('.ui.remote.selection.dropdown').dropdown('refresh');

`            $('.ui.remote.selection.dropdown')
            .dropdown({
              apiSettings: {
                url: '/data/get'
              },
              clearable: false,
              filterRemoteData: true,
              cache: false,
              saveRemoteData: false,
              onChange: function(value, text, $selectedItem) {
              }
            });`

**Comment by lubber-de** [comment, marked as answer]
Use the `query remote` behavior
```javascript
$('.ui.dropdown').dropdown('query remote','',function(){
  	$('body').toast({
    	message: 'Refreshed!'
    });
  });
```
See https://jsfiddle.net/lubber/8hyxegkL/

**Comment by hakeem86** [comment]
You are awesome.. where is this documented ?

**Reply by lubber-de** [reply]
It's not documented, because it was supposed to be used as an internal function only.
However, every method of each component can be called as a behavior.
